.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,79 @@
+name: "🐛 Bug Report"
+description: If you've found a reproducible bug
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
+
+        We're most likely to fix bugs that are easy to reproduce, so please try
+        to provide enough information for us to understand how to reproduce the
+        issue.
+
+  - type: textarea
+    attributes:
+      label: What is the issue?
+      description: |
+        A clear and concise description of what Everest is doing and what you
+        would expect.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: How can it be reproduced?
+      description:
+        A clear and concise description of how to reproduce the issue. If
+        possible, include a minimal example that can be run in a test
+        environment.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Logs, error output, etc
+      description: |
+        Please include any relevant logs, error output, or screenshots that
+        might help us understand the issue. If you have a lot of logs, please
+        consider using a service like [Gist](https://gist.github.com/) to share them.
+
+  - type: input
+    attributes:
+      label: Everest Version
+      description: What version of Everest are you running?
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Kubernetes Distribution
```

Comment:
is K8S version important for us?

Reply:
maybe, it depends. But good to know. That's why I've put it under the `Environment details` section as an optional field

Reply:
I think "Installation Method" can also be useful, in a dropdown

> options:
        - Helm
        - CLI
        - GitHub Release (manual YAML)
        - From Source
        - Other (please specify in Environment Details)

An Operating System can help as well:

> description: What OS are you using on the system running Everest?
      placeholder: e.g. Ubuntu 22.04, macOS 14.3, Windows 11 WSL2
      
I saw Airflow as example> https://github.com/apache/airflow/issues

Reply:
I agree, "Installation method" is a good idea. I've added a dedicated field for it. I'm not sure OS would be so useful to us -- so far we've not encountered issues that may be triggered due to OS. Nevertheless, they can add that in the "Environment details" section (I'v added a placeholder for it now).. thanks @edithturn